mscore/pluginCreator.cpp

```diff
@@ -240,20 +240,22 @@ void PluginCreator::readSettings()
 //---------------------------------------------------------
 
 void PluginCreator::closeEvent(QCloseEvent* ev)
-      {
+      {
       if (state == PCState::DIRTY) {
             QMessageBox::StandardButton n = QMessageBox::warning(this, tr("MuseScore"),
                tr("Plugin \"%1\" has changes.\n"
                "Save before closing?").arg(path),
                QMessageBox::Save | QMessageBox::Discard | QMessageBox::Cancel,
                QMessageBox::Save);
-            if (n == QMessageBox::Save)
-                  savePlugin();
+            if (n == QMessageBox::Save){
```

Comment:
Space missing between ) and {
But then again those curly braces are not need at all

Reply:
But from https://musescore.org/en/handbook/musescore-coding-rules
"Exception 2: Use braces also in if-then-else blocks where either the if-code or the else-code covers several lines:

if (address.isEmpty()) {
      --it;
      } 
else {
      qDebug("%s", qPrintable(address));
      ++it;
      }"
I will just do like you tell me

Reply:
okay the spaces didn't copy in previous post, sorry for that